Library/Formula/apache-drill.rb

```diff
@@ -0,0 +1,22 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                /usr/local/Library/Contributions/example-formula.rb
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class ApacheDrill < Formula
+  homepage ""
+  url "http://getdrill.org/drill/download/apache-drill-0.9.0.tar.gz"
+  version "0.9.0"
```

Comment:
This is redundant here. The URL is simple enough that `brew` can grab the version automatically.